### ISSUE TITLE
A constant constructor can't call a non-constant super constructor.

### ISSUE THREAD

**Issue by briouyahasnae** [syncfusion/flutter-widgets#1199]
When I'm trying to run my app I find this issue: 

> Task :app:compileFlutterBuildDebug ../../.pub-cache/hosted/pub.dev/syncfusion_flutter_core-21.2.5/lib/src/calendar/custom_looping_widget.dart:50:9: Error: A constant constructor can't call a non-constant super constructor. : super(children: children); ^ Target kernel_snapshot failed: Exception > Task :app:compileFlutterBuildDebug FAILED FAILURE: Build failed with an exception.

**Comment by badshamasala** [comment]
yes i also get this issue if you get the solution please let me know

**Comment by briouyahasnae** [comment]
@badshamasala for the moment I overrided the plugin 
syncfusion_flutter_core : 21.2.3 . I used this version 21.2.3 . and it works

**Comment by badshamasala** [comment]
still getting the issue in 21.2.3 whats your flutter version

**Comment by badshamasala** [comment]
../../../.pub-cache/hosted/pub.dev/syncfusion_flutter_charts-21.2.5/lib/src/common/template/rendering.dart:401:9: Error: A constant constructor can't call a non-constant super constructor.
      : super(children: children);
        ^
Target kernel_snapshot failed: Exception


FAILURE: Build failed with an exception.

* Where:
Script '/home/xyz/dev/flutter/packages/flutter_tools/gradle/flutter.gradle' line: 1151

* What went wrong:
Execution failed for task ':app:compileFlutterBuildDebug'.
> Process 'command '/home/xyz/dev/flutter/bin/flutter'' finished with non-zero exit value 1

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 6s
Running Gradle task 'assembleDebug'...                              6.9s
Exception: Gradle task assembleDebug failed with exit code 1

**Comment by briouyahasnae** [comment]
for pubspec.yaml use this
dependency_overrides:
  syncfusion_flutter_core: 21.1.41
and 
    **delete** pubspec.lock 
then 
    **run** flutter pub get

**Comment by Shisah** [comment]
version 21.1.41 has no issues

**Comment by badshamasala** [comment]
i am still getting the same issue since yesterday

**Comment by badshamasala** [comment]
can you please chech the package once again i tried both syncfusion_flutter_core: 21.1.41 syncfusion_flutter_charts: 21.1.41

**Comment by badshamasala** [comment]
but I am using syncfusion_flutter_charts
so why I would use CORE?

**Comment by leidig54** [comment]
Because the charts package use the core package, but the new core package is broken. Adding the dependency override forces syncfusion_flutter_charts to use a working version of syncfusion_flutter_core.

**Comment by nurindah-bp** [comment]
@Shisah it worked thanks

**Comment by IndumathiR-1995** [comment]
As per the shared information, we have checked the issue “A constant constructor can't call a non-constant super constructor” in a simple sample of the Flutter calendar, but we are unable to reproduce the mentioned issue, it was working fine as expected from our end.

Tested details:

1. Flutter version: Stable channel-3.10.0

2. Flutter  Calendar: 21.2.5


Kindly update and check the Flutter version to the latest version 3.10.0. and let us know still if you are facing the same issue, please share the below details with us.



- Replication procedure
- Code snippet of the calendar implementation.
- 

It would be helpful for us to analyze and provide you with a solution at the earliest.

**Comment by briouyahasnae** [comment]
I updated flutter version to 3.10.1 and Flutter Calendar: 21.2.5 and  syncfusion_flutter_calendar: 21.1.41 and it's work

**Comment by andersonribeir** [comment]
Fixed?
